crates/subspace-node/src/bin/subspace-node.rs

```diff
@@ -489,7 +492,50 @@ fn main() -> Result<(), Error> {
                         sc_service::Error::Other(format!(
                             "Failed to build a full subspace node: {error:?}"
                         ))
-                    })?
+                    })?;
+
+                    let mut xdm_gossip_worker_builder = GossipWorkerBuilder::new();
+                    let relayer_worker =
+                        domain_client_message_relayer::worker::relay_consensus_chain_messages(
```

Comment:
Considering the number of consensus nodes is larger than any single domain operator node and every consensus node will try to relay every XDM of the consensus that may be a large amount of duplicated XDM extrinsic and increase the network traffic, will it be a concern?

Reply:
Due to the probabilistic nature, it not possible to actually know a set of farmers that can can relay. Making the current block author do the relay means, we need other mechanism to ensure messages are not skipped. But doing that is more unncessary implemnetation without proper reward system. Since these farmers are gossiping over separate channel and they do not gossip of already seen, this is not a big concern for me yet.

Reply:
Should be updated where consensus relayer is only run when the node is running an operator